OIPA/api/transaction/serializers.py

```diff
@@ -310,10 +310,10 @@ def update(self, instance, validated_data):
         update_instance.id = instance.id
         update_instance.save()
 
+
         if provider_data.get('ref'):
-            provider_org = models.TransactionProvider.objects.create(
```

Comment:
@HatimKH please add the same check for receiver_org, then feel free to merge it yourself.